README.rst

```diff
@@ -828,6 +828,14 @@ This should end with a 'PASSED' result::
     PASSED (successes=143)
 
 
+To install the synapse homeserver from your source run:
+
+    source env/bin/activate
+    mkdir build && cd build
+    pip install ..
+
+This should install synapse in the `/build` directory
```

Comment:
`pip install` installs things in the virtualenv, not the current directory, so I'm not sure what you're doing with the build directory here.

I've never actually found this to be necessary - `synctl` should work without installing the synapse package.